src/status_im/contexts/wallet/common/asset_list/view.cljs

```diff
@@ -12,11 +12,11 @@
         fiat-value       (utils/total-token-fiat-value currency token)
         fiat-formatted   (utils/get-standard-fiat-format crypto-formatted currency-symbol fiat-value)]
     [quo/token-network
-     {:token       (:symbol token)
+     {:token       (keyword (:symbol token))
```

Comment:
we should make this has no side effects on it's previous uses. 
cc @briansztamfater / @OmarBasem

Reply:
I don't think it does

Reply:
I changed the schema to accept keyword and string